Weather/Modules/CityDetail/Builder/WeatherDetailSwiftInjectBuilder.swift

```diff
@@ -22,13 +22,13 @@ struct WeatherDetailSwiftInjectBuilder : WeatherDetailBuilder {
     }
     
     private func registerInteractor() {
-        Container.sharedContainer.register(WeatherDetailInteractor.self){ r in
+        Container.sharedContainer.register(WeatherDetailInteractor.self) { r in
```

Comment:
You are right! I've added SwiftLint to the project so we keep good practices on formatting!